starter-code/src/viking.js

```diff
@@ -1,11 +1,79 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+
+  this.attack = function () {
```

Comment:
The best practice is to use methods on prototypes so here should be something like:
```
Soldier.prototype.attack = function() {
  return this.strength;
};
```